_static/files/audit-event-examples/R4_read.json

```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://hl7.org/fhir/audit-event-type",
+        "code": "rest",
+        "display": "RESTful Operation"
+    },
+    "subtype": [
+        {
+            "system": "http://hl7.org/fhir/http-verb",
+            "code": "GET"
+        },
+        {
+            "system": "http://hl7.org/fhir/restful-interaction",
+            "code": "read"
+        }
+    ],
+    "action": "R",
+    "recorded": "2022-10-07T14:21:15.456+00:00",
+    "outcome": "0",
+    "outcomeDesc": "200",
+    "agent": [
+        {
+            "altId": "unknown",
+            "who": {
+                "display": "anonymous"
+            },
+            "type": {
+                "coding": [
+                    {
+                        "system": "https://profiles.ihe.net/ITI/BALP/CodeSystem/UserAgentTypes",
+                        "code": "UserOauthAgent",
+                        "display": "User OAuth Agent participant"
+                    }
+                ]
+            },
+            "requestor": true,
+            "network": {
+                "address": "127.0.0.1",
+                "type": "2"
+            }
+        },
+        {
+            "type": {
+                "coding": [
+                    {
+                        "system": "http://dicom.nema.org/resources/ontology/DCM",
+                        "code": "110152",
+                        "display": "Destination Role ID"
+                    }
+                ]
+            }
+        }
+    ],
+    "source": {
+        "site": "Firely Server",
+        "type": [
+            {
+                "system": "http://hl7.org/fhir/security-source-type",
+                "code": "4",
+                "display": "Application Server"
+            }
+        ],
+        "observer": {
+            "identifier": {
+                "system": "http://vonk.fire.ly/fhir/sid/devices",
+                "value": "firely-server"
+            },
+            "display": "SERVER_HOST_NAME"
+        }
+    },
+    "entity": [
+        {
+            "what": {
+                "identifier": {
+                    "value": "REQUEST_ID"
+                },
+                "type": "Device"
+            },
+            "type": {
+                "system": "http://fire.ly/fhir/sid/api-diagnostics",
```

Comment:
Here we should use http://terminology.hl7.org/CodeSystem/audit-entity-type as it's an extensible binding in the IHE IG

Reply:
Don't show this information is Auth isn't used

Reply:
Let's discuss it whenever you have time